packages/console/src/pages/SignInExperience/components/ManageLanguageModal/AddLanguageSelector.module.scss

```diff
@@ -0,0 +1,24 @@
+@use '@/scss/underscore' as _;
+
+.addLanguageButton {
+  width: 100%;
+  border-color: var(--color-outline);
+  color: var(--color-text);
+  background-color: unset;
```

Comment:
just curious, why do we need an unset prop here, since didn't find any parent element related specific styles?

Reply:
Because the `<Button />` component with an 'outline' style has a default white background color, unset for we don't need the background color in this case.

Reply:
background none?

Reply:
updated.